crates/flox/src/utils/init/gitConfig.in

```diff
@@ -3,28 +3,9 @@
 	name = Flox User
 	email = floxuser@example.invalid
 
-# Use only https for accessing flox utility libraries.
-[url "https://github.com/flox/capacitor"]
-	insteadOf = "ssh://git@github.com/flox/capacitor"
-	insteadOf = "git@github.com:flox/capacitor"
+[credential "https://git.saas.floxdev.com/"]
+	helper = "!{flox_gh_bin} auth token --git-credential-helper"
 
-[url "https://github.com/flox/nixpkgs-flox"]
-	insteadOf = "ssh://git@github.com/flox/nixpkgs-flox"
-	insteadOf = "git@github.com:flox/nixpkgs-flox"
-
-[url "https://github.com/flox/nixpkgs-catalog"]
-	insteadOf = "ssh://git@github.com/flox/nixpkgs-catalog"
-	insteadOf = "git@github.com:flox/nixpkgs-catalog"
-
-[url "https://github.com/flox/catalog-ingest"]
-	insteadOf = "ssh://git@github.com/flox/catalog-ingest"
-	insteadOf = "git@github.com:flox/catalog-ingest"
-
-[url "https://github.com/flox/flox-extras"]
-	insteadOf = "ssh://git@github.com/flox/flox-extras"
-	insteadOf = "git@github.com:flox/flox-extras"
-
-[url "https://github.com/flox/bundlers"]
-	insteadOf = "ssh://git@github.com/flox/bundlers"
-	insteadOf = "git@github.com:flox/bundlers"
-{original_include}
+# Temporary while we transition to flox gitforge.
+[credential "https://github.com/"]
+	helper = "!{gh_bin} auth git-credential"
```

Comment:
why do we use `git-credential` here instead of `token --git-credential-helper`?

Reply:
Good question. `gh auth git-credential` is a feature offered by the GitHub CLI that works by receiving input on STDIN identifying the requested hostname (i.e. github.com), and it uses that to look up the token in the `hosts.yml` file.

This approach won't work for the flox CLI authentication because the requested [git] hostname will be a flox one, but the authentication hostname is always github.com - i.e. they don't match. That's why we've extended the existing `gh auth token` capability to print out the username and token in "git credential helper" format, and added the new arg accordingly. Another way of looking at it is to note that `gh auth token --git-credential-helper` isn't a supported argument; that argument is only supported by `flox-gh`.

Hope that makes sense!